packages/plugins/FileService/src/SNSAdaptor/index.tsx

```diff
@@ -89,13 +95,55 @@ const definition: Plugin.SNSAdaptor.Definition = {
 
 export default definition
 
-function onAttachedFile(payload: FileInfo) {
-    const name = truncate(payload.name, { length: 10 })
+function openDialog(props: Partial<FileServiceDialogProps>) {
+    CrossIsolationMessages.events.compositionDialogEvent.sendToLocal({
+        reason: 'timeline',
+        open: true,
+        options: {
+            startupPlugin: base.ID,
+            startupPluginProps: props,
+            isOpenFromApplicationBoard: false,
+        },
+    })
+}
+
+function onAttachedFile(file: FileInfo): Plugin.SNSAdaptor.BadgeDescriptor {
+    const name = truncate(file.name, { length: 10 })
+    const size = formatFileSize(file.size)
+
+    return {
+        text: (
+            <div
```

Comment:
Did you forget to use a `<Typography />` component?

Reply:
Fixed.